rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,501 @@
+code changes and their impact.
+
+| Component/Deliverable                                 	| Responsible Team/Service 	| No earlier than 	| Contact Person	|
+|-------------------------------------------------------	|--------------------------	|-----------------	|---------------	|
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis Pipelines                                    	| Broad                    	| Q2 2019         	| ?             	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| TBD             	| S. Ehsan      	|
+| Data Analysis Service starfish Pipeline Tests         	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| Z. Perova             	|
+| [Ability to describe files that belong together][a]   	| ?                        	| ?               	| ?             	|
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| ?             	|
+| DSS spatial data bundle (or bundless) specification   	| UCSC                     	| ?               	| B. Hannafious 	|
+| DSS scale testing                                     	| UCSC / DCP               	| TBD             	| B. Hannafious 	|
+| Data Browser facets for spatial data                  	| UCSC                     	| TBD             	| H. Schmidt    	|
+| Matrix service integration                            	| starfish                 	| TBD             	| M. Kinsella   	|
+| Upload Bundle for 1M+ files                           	| ?                        	| ?               	| P. Shah       	|
+| Image file download latency testing                   	| ?                        	| ?               	| ?             	|
+
+### **Unresolved Questions**
+
+  - Who are the intended beta users for imaging support and what
+    functionality is being targeted?
+
+  - Are there other roadblocks for imaging support in the DCP?
```

Comment:
We need to have a clearly defined process for wrangling image-based transcriptomics datasets.

Reply:
Good point. What's the right next step for this? An issue in a repository owned by the wranglers? Where have you defined the process for wrangling sequencing data?

Reply:
@ambrosejcarr we have documentation in the wrangling repo on the process of wrangling datasets in general. I also shared this [Google doc](https://docs.google.com/document/d/1pc2h7pv5ayDm8VOxu2aN3OrEtQtXi_v6ifZKDxp4-_Y/edit) adapted to wrangling image-based transcriptomics datasets that require SpaceTx formatting. I will be very happy to hear feedback from the involved/mentioned parties.